src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -8,6 +8,7 @@ public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_VOLUNTEER_NRIC = "The volunteer NRIC provided is invalid";
```

Comment:
Refer to https://github.com/AY2223S2-CS2103T-W12-1/tp/pull/69#discussion_r1119903070

Reply:
Yup that makes sense. Changed it.